packages/components/src/styles/copyright/copyright.css

```diff
@@ -17,4 +16,27 @@
       }
     }
   }
+
+  &--color {
+    &-black {
+      &::after {
+        @apply text-black;
+      }
+    }
+  }
+
+  &--shadow {
+    &::after {
+      @apply text-white;
+      text-shadow: 0.5px 0.5px 1.5px #515151;
```

Comment:
Looking at Figma I feel this should probably be a drop-shadow. There is a new token `shadow-sm` with the description `Used for copyright text` but it's defined as `box-shadow`so it doesn't work in this case, we would need to update the token and also make some changes in `create-theme`.
@coraliefeil @mariohamann what do you think?

Reply:
Updating create-theme sounds reasonable.

Reply:
Updated